src/app/content/highlights/components/cardUtils.ts

```diff
@@ -94,6 +107,11 @@ const updateStackedCardsPositions = (
   for (const [index, highlight] of highlightsElements.entries()) {
     const topOffset = getHighlightPosition(highlight).top;
 
+    if (topOffset === hiddenHighlightOffset) {
+      positions.set(highlight.id, topOffset);
```

Comment:
Yes, that was actually my idea, but now I think it has one disadvantage - it will be confusing when using a keyboard to navigate between cards. We should probably hide it also with CSS

Reply:
Yup, it shows it offscreen and it prevents changing the position through relationships among others highlights.
The side effect is that opening the collapsed wrapper makes the hidden highlights to fall from the top of the screen.

Reply:
> We should probably hide it also with CSS

that could be the solution

Reply:
i would prefer that these cards be hide/shown and have the same animation coming in and out that they normally do